packages/luna/src/components/Toast/RadixToast.tsx

```diff
@@ -0,0 +1,77 @@
+import { createContext, useContext, useState } from "react";
+import * as Toast from "@radix-ui/react-toast";
+import cn from "classnames";
+
+type ToastType = "success" | "failed";
```

Comment:
Also How about if you also exported this as an enum, as this is a general component that can be used everywhere. so if we added or changed one of the types, then you won't need to change it everywhere it's used again.
You can look at this [example](https://github.com/litespace-org/litespace/blob/e7e13b8b7ac2288aedd57fef5a27cf41f7371f4e/packages/luna/src/components/Button/types.ts#L1)